magefiles/target/sync-chart-md.go

```diff
@@ -2,23 +2,22 @@ package target
 
 import (
 	"fmt"
+	"github.com/tidwall/gjson"
 	"io"
 	"net/http"
 	"os"
 	"strings"
 
+	"botkube.io/tools/printer"
 	"github.com/MakeNowJust/heredoc/v2"
 	"github.com/samber/lo"
-	"github.com/tidwall/gjson"
-
-	"botkube.io/tools/printer"
```

Comment:
**nit** (optional) we can revert the import grouping :)

Reply:
done